executors/realesrgan/executor.py

```diff
@@ -0,0 +1,242 @@
+import enum
+import time
+
+from PIL import Image
+from io import BytesIO
+from pathlib import Path
+from typing import Any, Dict, List
+from urllib.request import urlopen
+
+import numpy as np
+
+from basicsr.archs.rrdbnet_arch import RRDBNet
+from basicsr.utils.download_util import load_file_from_url
+from gfpgan import GFPGANer
+from jina import Executor, DocumentArray, Document, requests
+from realesrgan import RealESRGANer
+from realesrgan.archs.srvgg_arch import SRVGGNetCompact
+
+
+GFPGAN_MODEL_NAME = 'GFPGANv1.3.pth'
+GFPGAN_MODEL_URL = 'https://github.com/TencentARC/GFPGAN/releases/download/v1.3.0/GFPGANv1.3.pth'
+
+
+class RESRGAN_MODELS(str, enum.Enum):
+    RealESRGAN_x4plus = 'RealESRGAN_x4plus'
+    RealESRNet_x4plus = 'RealESRNet_x4plus'
+    RealESRGAN_x4plus_anime_6B = 'RealESRGAN_x4plus_anime_6B'
+    RealESRGAN_x2plus = 'RealESRGAN_x2plus'
+    RealESR_animevideov3 = 'realesr-animevideov3'
+    RealESR_general_x4v3 = 'realesr-general-x4v3'
+
+
+class RealESRGANUpscaler(Executor):
+    # resrgan_models is organized as:
+    # {
+    #   model_name: {
+    #     'name': str,
+    #     'netscale': int, (scaling strength eg 4=4x)
+    #     'model': initialized RealESRGAN model,
+    #     'model_face_fix': initialized GFPGAN model, [optional, non-anime only]
+    #   }
+    # }
```

Comment:
if not needed lets remove

Reply:
It's a comment that explains how the `resrgan_models` dict entries are defined, so it's useful to anyone programming within the module.